### PR TITLE
Refactored the code

### PR DESCRIPTION
**Pull request by zf-development**
This is not needed as the player is already spawned on the playerLoaded event called.

### CODE REVIEW

**Comment by FjamZoo** [comment]
The reason why I added the 20 second delay to begin with is that if it tries to spawn the items before a player skin is FULLY loaded the items on their back might glitch out by doing random shit in fivem code, the solution would be to have some sort of checker like while not DoesEntityExsist(PlayerPedID()) do Wait(25) end

Not really THE solution but that is WHY the delay is there might be a good idea to look into something like that if u want to fully remove the wait

**Comment by zf-development** [comment]
For the `loadmodel()` and `requestAnimDict()`, I know that QBCore already have it but I didn't removed it in case someone is not up to date with the core but should be use..

**Comment by FjamZoo** [comment]
Ideally yes that would make the code look cleaner BUT when somehting like that is used very frequent you do lose performance by calling an export rather than calling a localized function so I'd rather keep it all in the file

If you can fix the merge conflicts then I can merge this thank you.

**Comment by zf-development** [comment]
@FjamZoo github is saying that there is a conflict.

when I look at the code, there is not conflicts at all. Not sure what's going on... If you can give me a hint on it so I can fix it.

File: client/main.lua [hunk 1]
```diff
@@ -1,583 +1,562 @@
...
-    end
-
-    ClearPedTasks(ped)
-  end)
-end
+    if carryingBox then return end -- Only allow the function to be run once at a time
+    carryingBox = item
+    local ped = PlayerPedId()
+    local dict, anim = props[item].dict or 'anim@heists@box_carry@', props[item].anim or 'idle'
+    if anim == 'none' then return end
```
Comment (uShifty) [inline client/main.lua:222]: Only conflict I can really see is that you didnt change the if statement in doAnim to ` if not anim or not dict then return end`

Comment (uShifty) [reply client/main.lua:222]: oh neverminded that wouldnt make sense, not sure 😂

**Pull request closed without merging**